document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -47,7 +47,8 @@ public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequ
         try {
             transaction = transactionService.getTransaction(resourceId);
         } catch (ServiceException e) {
-            LOG.error(e, debugMap);
+            LOG.errorContext("An error occurred when calling the transaction service with resource id: "
```

Comment:
The first argument is the context:
```
public void errorContext(String context, Exception e, Map<String, Object> data)
```

This method signature would support the arguments:
```
public void errorContext(String context, String message, Exception e, Map<String, Object> data)
```